src/components/PhotoPage/AdminApproval/MultiFields.js

```diff
@@ -0,0 +1,212 @@
+import React from "react";
```

Comment:
I was unable to commit this file without running prettier over it so has been caught up